packages/usage-reporting-protobuf/package.json

```diff
@@ -5,19 +5,24 @@
   "type": "module",
   "exports": {
     ".": {
-      "import": "./generated/protobuf.mjs",
-      "require": "./generated/protobuf.cjs",
-      "types": "./generated/protobuf.d.ts"
+      "import": "./generated/esm/protobuf.js",
+      "require": "./generated/cjs/protobuf.js",
+      "types": {
```

Comment:
Looks like we had this incorrect previously. The `types` key should come first in this object. Funny little quirk. Small note about it in the docs if you scroll down from here:
https://www.typescriptlang.org/docs/handbook/esm-node.html#packagejson-exports-imports-and-self-referencing

Reply:
Interesting, I didn't know that